04-user-guide.Rmd

```diff
@@ -7,25 +7,15 @@ guide](#developer-installation-guide).
 ## User Installation Guide
 This section describes how to install the FIMS R package and dependencies.
 
-## Installing the package from Github
+## Installing the package from R universe
 
 The following software is required:
-- R version 4.0.0 or newer (or RStudio version 1.2.5042 or newer)
-  - the `remotes` R package
-- `TMB` (install instructions at are
-[here](https://github.com/kaskr/adcomp/wiki/Download).)
+- R version 4.0.0 or newer
 
-### Windows users
-
-- Rtools4.4 (available from [here](https://cran.r-project.org/bin/windows/Rtools/))
-  - this likely requires IT support to install it on NOAA computers
-  (or any without administrative accounts)
-
-## Installing from `R`
+Install the latest [FIMS from R universe](https://noaa-fims.r-universe.dev/FIMS) within R:
```

Comment:
add "to install a precompiled version of the package" so that it is clear why you would use this version rather than the version on the README of NOAA-FIMS/FIMS.